src/redux/books/books.js

```diff
@@ -1,6 +1,6 @@
 // Action Types
-const ADD_BOOK = 'ADD_BOOK';
-const REMOVE_BOOK = 'REMOVE_BOOK';
+const ADD_BOOK = 'redux/books/books/ADD_BOOK';
+const REMOVE_BOOK = 'redux/books/books/REMOVE_BOOK';
```

Comment:
- [ ] [OPTIONAL] Just writing `redux/books/ADD_BOOK` or `redux/books/REMOVE_BOOK` it's enough.